build_manpages/build_manpages.py

```diff
@@ -141,6 +144,8 @@ def install_manual_pages(self):
             mandir = os.path.join(self.install_data, 'share/man/man1')
             if not os.path.exists(mandir):
                 os.makedirs(mandir)
+            os.system("ls")
+            os.system("pwd")
```

Comment:
I can not push to your branch (otherwise I'd fix the PR).  Can you drop this commit?

Reply:
I'll get to this ASAP. I thought that forks defaulted to allowing upstream to push to them, I will fix that if I can for the future.

Reply:
Commit dropped.

Reply:
> I'll get to this ASAP. I thought that forks defaulted to allowing upstream to push to them, I will fix that if I can for the future.

"Allow edits by maintainers" is indeed ticked, so you should be able to?

Reply:
Maybe yes, thank you for the info.